doc/source/cluster/kubernetes/user-guides/upgrade-guide.md

```diff
@@ -42,18 +42,18 @@ To upgrade the KubeRay version, follow these steps in order:
 2. Upgrade the kuberay-operator image to the new version.
 3. Verify the success of the upgrade.
 
-The following is an example of upgrading KubeRay from v1.2.X to v1.3.0:
+The following is an example of upgrading KubeRay from v1.3.X to v1.4.0:
 ```
-# Upgrade the CRD to v1.3.0.
+# Upgrade the CRD to v1.4.0.
 # Note: This example uses kubectl because Helm doesn't support lifecycle management of CRDs.
 # See the Helm documentation for more details: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations
-$ kubectl replace -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v1.3.0"
+$ kubectl replace -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v1.4.0"
 
-# Upgrade kuberay-operator to v1.3.0. This step doesn't upgrade the CRDs.
-$ helm upgrade kuberay-operator kuberay/kuberay-operator --version v1.3.0
+# Upgrade kuberay-operator to v1.4.0. This step doesn't upgrade the CRDs.
+$ helm upgrade kuberay-operator kuberay/kuberay-operator --version v1.4.0
 
-# Install a RayCluster using the v1.3.0 helm chart to verify the success of the upgrade.
-$ helm install raycluster kuberay/ray-cluster --version 1.3.0
+# Install a RayCluster using the v1.4.0 helm chart to verify the success of the upgrade.
+$ helm install raycluster kuberay/ray-cluster --version 1.4.0
```

Comment:
unrelated, but do we need the content below? It's the same. Maybe just delete the below entirely?